cluster/addons/dns/skydns-rc.yaml.in

```diff
@@ -19,6 +19,9 @@ spec:
         version: v11
         kubernetes.io/cluster-service: "true"
     spec:
+{% if grains['cloud'] is defined and grains['cloud'] in [ 'vsphere', 'photon-controller' ] %}
```

Comment:
I don't disagree that it's a bad situation, but it is what it is.  This
file is processed by other scripts that are not using jinja to do the
expansion.  We'll need to find a better way to do this (if indeed we want
to do this)

On Sun, May 1, 2016 at 2:50 PM, Mike Danese <notifications@github.com>
wrote:

> In cluster/addons/dns/skydns-rc.yaml.in
> <https://github.com/kubernetes/kubernetes/pull/24124#discussion_r61693688>
> :
>
> > @@ -19,6 +19,9 @@ spec:
> >          version: v11
> >          kubernetes.io/cluster-service: "true"
> >      spec:
> > +{% if grains['cloud'] is defined and grains['cloud'] in [ 'vsphere', 'photon-controller' ] %}
>
> I'm going to rollback the changes to this single file. @thockin
> <https://github.com/thockin> Why was this not caught in CI? We have
> multiple DNS e2e tests. I also pinged the cluster team for review and got
> no response. I'm not sure what more we could have done to avoid this.
>
> This change adds valid jinja to a jinja template. It's unreasonable that
> we can't make a change like this locally.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/24124/files/5699d51efadb41b6c010f45672c4844d84de0b84#r61693688>
>